src/main/java/com/faforever/client/chat/UserInfoWindowController.java

```diff
@@ -232,6 +258,11 @@ public void setPlayer(Player player) {
         });
   }
 
+  private void updateNameHistory(Player player) {
+    fafService.getPlayersByIds(Collections.singletonList(player.getId()))
+        .thenAccept(players -> nameHistoryTable.setItems(players.get(0).getNames()));
```

Comment:
We should handle `.exceptionally()` as well. You could use `com.faforever.client.notification.NotificationService#addImmediateErrorNotification`